src/sparc/client/services/_default.py

```diff
@@ -0,0 +1,28 @@
+import os
+from abc import ABC, abstractmethod
+
+
+class ServiceBase(ABC):
+    @abstractmethod
+    def __init__(self, connect, *args, **kwargs) -> None:
```

Comment:
For the interface, I would suggest to annotate nicely all explicit arguments, for instance
```python
 def __init__(self, connect: bool, *args, **kwargs) -> None:

```

Reply:
Done